### PR TITLE
Fix: Remove scripts

### PR DESCRIPTION
**Pull request by localheinz**
### What is the reason for this PR?

- [x] removes scripts

Follows #773.

### Author's checklist

- [x] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)

### Review checklist

- [x] All checks have passed
- [ ] Changes are added to the `CHANGELOG.md`
- [x] Changes are approved by maintainer

### CODE REVIEW

File: bin/documentor.php
Comment (localheinz) [inline bin/documentor.php:1]: Running

```shell
php test/documentor.php
```

on current `2.0` yields

```
PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function Faker\Provider\en_US\Payment::calculateRoutingNumberChecksum(), 0 passed and exactly 1 expected in /Users/am/Sites/FakerPHP/Faker/src/Provider/en_US/Payment.php:28
Stack trace:
#0 [internal function]: Faker\Provider\en_US\Payment::calculateRoutingNumberChecksum()
#1 /Users/am/Sites/FakerPHP/Faker/src/Generator.php(696): call_user_func_array(Array, Array)
#2 /Users/am/Sites/FakerPHP/Faker/src/Documentor.php(50): Faker\Generator->format('calculateRoutin...')
#3 /Users/am/Sites/FakerPHP/Faker/test/documentor.php(8): Faker\Documentor->getFormatters()
#4 {main}
  thrown in /Users/am/Sites/FakerPHP/Faker/src/Provider/en_US/Payment.php on line 28

Fatal error: Uncaught ArgumentCountError: Too few arguments to function Faker\Provider\en_US\Payment::calculateRoutingNumberChecksum(), 0 passed and exactly 1 expected in /Users/am/Sites/FakerPHP/Faker/src/Provider/en_US/Payment.php:28
Stack trace:
#0 [internal function]: Faker\Provider\en_US\Payment::calculateRoutingNumberChecksum()
#1 /Users/am/Sites/FakerPHP/Faker/src/Generator.php(696): call_user_func_array(Array, Array)
#2 /Users/am/Sites/FakerPHP/Faker/src/Documentor.php(50): Faker\Generator->format('calculateRoutin...')
#3 /Users/am/Sites/FakerPHP/Faker/test/documentor.php(8): Faker\Documentor->getFormatters()
#4 {main}
  thrown in /Users/am/Sites/FakerPHP/Faker/src/Provider/en_US/Payment.php on line 28
```

Running

```shell
git grep documentor\.php
```

yields nothing.

Perhaps we do not need it anymore?

File: example/test.php
Comment (localheinz) [inline example/test.php:1]: Running

```shell
php test/test.php
```

on current `2.0` yields

```xml
<?xml version="1.0" encoding="UTF-8"?><contacts>
  <contact firstName="Alvina" lastName="Champlin" email="xhettinger@wuckert.com" >
    <phone number="+1 (559) 579-1877"/>
    <birth date="2015-03-23" place="Carolineview"/>
    <address>
      <street>27643 Goldner Fall Apt. 119</street>
      <city>North Jeanette</city>
      <postcode>51335</postcode>
      <state>New Jersey</state>
    </address>
    <company name="Rempel, Ankunding and Ratke" catchPhrase="Universal dedicated securedline">
      <offer>generate cutting-edge partnerships</offer>
      <director name="Mrs. Iva Baumbach Jr." />
    </company>
  </contact>
  <contact firstName="Alexander" lastName="Krajcik" email="brisa22@yahoo.com" >
    <phone number="+1-607-583-9256"/>
    <birth date="2023-09-04" place="Pearlborough"/>
    <address>
      <street>392 Kautzer Mountain</street>
      <city>Arthurstad</city>
      <postcode>18272</postcode>
      <state>Kentucky</state>
    </address>
    <company name="Huels-Feeney" catchPhrase="Secured homogeneous structure">
      <director name="Maximillian Tillman" />
    </company>
  </contact>
  <contact firstName="Turner" lastName="Bashirian" email="reilly94@mann.biz" >
    <phone number="680.971.0779"/>
    <birth date="1992-02-23" place="South Kameron"/>
    <address>
      <street>820 Raul Summit Suite 353</street>
      <city>Collierville</city>
      <postcode>35209</postcode>
      <state>North Dakota</state>
    </address>
    <company name="Will-Kulas" catchPhrase="Enhanced logistical success">
      <offer>streamline collaborative markets</offer>
      <director name="Clair Ziemann" />
    </company>
  </contact>
  <contact firstName="Bradly" lastName="Kshlerin" email="lupe96@wisoky.biz" >
    <phone number="+1 (901) 742-8499"/>
    <birth date="1975-03-14" place="South Cecelia"/>
    <address>
      <street>648 Medhurst Forge</street>
      <city>Roobville</city>
      <postcode>95031-1610</postcode>
      <state>Illinois</state>
    </address>
    <company name="Yundt LLC" catchPhrase="Inverse neutral initiative">
    </company>
  </contact>
  <contact firstName="Taya" lastName="Sauer" email="jakubowski.anya@hotmail.com" >
    <phone number="+18307281280"/>
    <address>
      <street>760 Araceli Greens</street>
      <city>Guadalupetown</city>
      <postcode>37371-4514</postcode>
      <state>Nevada</state>
    </address>
    <company name="Hermiston, Runolfsdottir and Hayes" catchPhrase="Enhanced bottom-line model">
      <director name="Prof. Lorine Lubowitz" />
    </company>
  </contact>
  <contact firstName="Maria" lastName="Ritchie" email="flatley.elfrieda@gmail.com" >
    <phone number="+1 (351) 889-8102"/>
    <birth date="1982-08-30" place="Port Amaliaville"/>
    <address>
      <street>6570 Ritchie Stream Apt. 337</street>
      <city>Port Ashleetown</city>
      <postcode>06269-2564</postcode>
      <state>Mississippi</state>
    </address>
    <company name="Heathcote, Weber and Fisher" catchPhrase="Exclusive 4thgeneration function">
      <director name="Layla Trantow" />
    </company>
  </contact>
  <contact firstName="Bobbie" lastName="McDermott" email="jalen.durgan@bartoletti.biz" >
    <phone number="(914) 617-5284"/>
    <birth date="1934-08-22" place="Prosaccoberg"/>
    <address>
      <street>4859 Bo Islands Suite 518</street>
      <city>Kleinton</city>
      <postcode>09912-8498</postcode>
      <state>Utah</state>
    </address>
    <company name="Gusikowski Inc" catchPhrase="Customer-focused tangible extranet">
    </company>
  </contact>
  <contact firstName="Maryam" lastName="Adams" email="kiara46@nitzsche.com" >
    <phone number="934-566-5343"/>
    <address>
      <street>7177 Rolfson Locks</street>
      <city>Konopelskifurt</city>
      <postcode>01284</postcode>
      <state>Ohio</state>
    </address>
    <company name="Nader, Welch and McKenzie" catchPhrase="Re-contextualized intangible focusgroup">
    </company>
  </contact>
  <contact firstName="Joanny" lastName="Hoeger" email="xward@pfeffer.com" >
    <phone number="+1-585-481-6414"/>
    <address>
      <street>12329 Bryce Summit</street>
      <city>New Eldoramouth</city>
      <postcode>23495</postcode>
      <state>New Hampshire</state>
    </address>
    <company name="Wolff, Dickinson and O'Keefe" catchPhrase="Sharable reciprocal capability">
    </company>
  </contact>
  <contact firstName="Christine" lastName="Sporer" email="schimmel.vada@yahoo.com" >
    <phone number="859.664.5065"/>
    <address>
      <street>174 Volkman Extensions</street>
      <city>Uptonton</city>
      <postcode>98268-3971</postcode>
      <state>Nevada</state>
    </address>
    <company name="Strosin, Wiza and Schmitt" catchPhrase="Stand-alone stable challenge">
      <director name="Sherman Cormier" />
    </company>
    <details>
<![CDATA[
Harum omnis fugit molestias veniam modi ut consequatur. Voluptates unde nesciunt impedit. Consectetur delectus aut repellat commodi vel itaque. Saepe nostrum eius nihil incidunt sunt amet perferendis. Voluptate et necessitatibus facere. Aperiam non libero est ut aliquid.]]>
    </details>
  </contact>
</contacts>
```

Perhaps we do not need it anymore?

Comment (GrahamCampbell) [reply example/test.php:1]: Yeh, probably we should just ditch this file.

**Comment by pimjansen** [comment]
Agree that we should just ditch both files though

**Review by bram-pkg — APPROVED** [review]

**Comment by localheinz** [comment]
Thank you, @bram-pkg, @GrahamCampbell, and @pimjansen!

**Pull request merged**